app/distributions/tests/test_api.py

```diff
@@ -0,0 +1,764 @@
+import datetime
+from unittest import mock
+
+from distributions.api import attribution_to_response
+from distributions.api import dataset_to_response
+from distributions.api import router
+from distributions.models import Attribution
+from distributions.models import Dataset
+from distributions.schemas import AttributionSchema
+from distributions.schemas import DatasetSchema
+from ninja.testing import TestClient
+from provider.models import Provider
+from schemas import TranslationsSchema
+
+from django.test import TestCase
+
+
+class ApiTestCase(TestCase):
+
+    def test_attribution_to_response_returns_response_with_language_as_defined(self):
+        provider = Provider.objects.create()
+        model_fields = {
+            "name_de": "BAFU",
+            "name_fr": "OFEV",
+            "name_en": "FOEN",
+            "name_it": "UFAM",
+            "name_rm": "UFAM",
+            "description_de": "Bundesamt für Umwelt",
+            "description_fr": "Office fédéral de l'environnement",
+            "description_en": "Federal Office for the Environment",
+            "description_it": "Ufficio federale dell'ambiente",
+            "description_rm": "Uffizi federal per l'ambient",
+            "provider": provider,
+        }
+        Attribution.objects.create(**model_fields)
```

Comment:
> Why not just `model = Attribution.objects.create(**model_fields)`?

Totally, that would be easier. Will fix.

> Also you might consider adding this to a setup function to avoid duplication of code (and also split test cases to endpoints)

Good point, I had also thought about it. I am not a fan of fixtures because I cannot see at a glance what the test does. Sometimes it's still a benefit when the setup is very long. So let me have another look, if it's duplicated often I can adapt it.